crates/turborepo-lib/src/commands/ls.rs

```diff
@@ -114,6 +148,21 @@ impl<'a> RepositoryDetails<'a> {
         for (package_name, entry) in &self.packages {
             println!("  {} {}", package_name, GREY.apply_to(entry));
         }
+    }
+
+    fn json_print(&self) {
+        println!("{}", serde_json::to_string_pretty(&self).unwrap());
```

Comment:
[praise](https://conventionalcomments.org/): I appreciate the JSON being pretty printed in a context like this.  thanks for thinking to do that!  it's always annoying when tools forget this (or, are against doing it, even for non perf-critical things like this API).